test/common/api_utils.py

```diff
@@ -0,0 +1,206 @@
+from on_http_api2_0 import ApiApi as Api
+from config.api2_0_config import config
+from json import loads
+import sys
+import os
+
+
+
+def api_node_select_argsList(node_type='compute', validate_obm=False):
+    """
+    This routine produces a list of node Ids that satisfy the requested parameters. 
+    Nodes may be filtered by Node Id,  SKU name, OBM MAC address which are defined
+    in the system environment.  The node type and OBM setting may be specified.
+    :param  node_type: Node type (i.e. compute) (default= 'compute')
+    :param  validate_obm: should the node have OBM settings (default=False)
+    return: A list with node IDs that match node type, SKU, and possible valid OBM settings.
+    """
+    return api_node_select(config.api_client,
+                           node_id=os.getenv("node_id", "None"),
+                           sku_name=os.getenv("SKU", "all"),
```

Comment:
The os.environ behavior of RackHD tests is changing.  There won't be things like SKU passed in the environment.  We do have an arg_list that will be available through the config block.  The change as you have it is fine for today but will need to updated shortly.